classes/Address.php

```diff
@@ -104,9 +104,13 @@ class AddressCore extends ObjectModel
 
     /** @var array Zone IDs cache */
     protected static $_idZones = [];
+
     /** @var array Country IDs cache */
     protected static $_idCountries = [];
 
+    /** @var array<int, bool> Store if an adress ID exists */
+    protected static $addressExists = [];
```

Comment:
You need to clean this in the `resetStaticCache` method or it will potentially mess with the tests

Reply:
Correct. Thanks